Source/Script/CRTErrorGenerator.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import AwsCommonRuntimeKit
+import AwsCCommon
+#if os(Linux)
+import Glibc
+#else
+import Darwin
+#endif
+
+extension FileHandle {
+    func writeln(_ str: String = "") {
+        self.write(str.data(using: .utf8)!)
+        self.write("\n".data(using: .utf8)!)
+    }
+
+    func writeTab() {
+        self.write("    ".data(using: .utf8)!)
+    }
+}
+
+@main
+struct CRTErrorGenerator {
```

Comment:
Is there any way this can be tested?

Reply:
I am not sure if we can write unit tests. I have tested it by brute forcing from 0 to Int32.max and making sure they both generate equal number of cases.